style.css

```diff
@@ -1,4 +1,347 @@
-* {
-  padding: 0;
-  margin: 0;
-}
+    * {
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+    }
+    
+    #pagemidcont {
+        display: flex;
+        justify-content: space-between;
+        width: 930px;
+        margin: 20px auto;
+    }
+    
+    #topicscont {
+        display: flex;
+        flex-direction: column;
+    }
+    
+    #asidepartcont {
+        display: flex;
+        flex-direction: column;
+    }
+    
+    .topic1 {
+        width: 610px;
+        height: 600px;
+    }
+    
+    .newsletter,
+    .category,
+    .popup1,
+    .popup2 {
+        background-color: #f9f9ff;
+        font-family: 'Lora', serif;
+        font-size: 20px;
+        position: relative;
+    }
+    
+    .catsize {
+        width: 290px;
+        height: 517px;
+    }
+    
+    .newslsize {
+        width: 290px;
+        height: 257px;
+    }
+    
+    .email {
+        width: 230px;
+        height: 50px;
+        padding: 0px 15px;
+        border-color: rgba(112, 111, 135, 0.2);
+        border: 1px solid #ced4da;
+    }
+    
+    .butn {
+        width: 230px;
+        height: 47px;
+        margin-bottom: 30px;
+        background-color: #222;
+        color: #ff9907;
+    }
+    
+    .butn:hover {
+        background-color: rgb(71, 63, 63);
+        color: #ff9907;
+    }
+    
+    .btnhvr {
+        transition: all 0.3s ease 0s;
+    }
+    
+    .btnhvr:hover {
+        background-color: #ff9907;
+        color: #222;
+        border: 0px transparent;
+    }
+    
+    .boxcont,
+    .boxcont2,
+    .boxcont3,
+    .boxcont4 {
```

Comment:
the same comment for these calsses. 
`.boxcont2' you can keep it.

Reply:
okay